components/cam/src/chemistry/modal_aero/aero_model.F90

```diff
@@ -2919,6 +2952,12 @@ subroutine modal_aero_depvel_part( ncol, t, pmid, ram1, fv, vlc_dry, vlc_trb, vl
                   gravit*slp_crc(i,k) / vsc_dyn_atm(i,k) ![m s-1] Stokes' settling velocity SeP97 p. 466
           vlc_grv(i,k) = vlc_grv(i,k) * dispersion
 
+          ! in the MMF NaN's were occurring here but the root cause was not
+          ! identified, so this check was added to work around the issue
+          if (use_MMF) then
+            if ( ieee_is_nan(vlc_grv(i,k)) ) vlc_grv(i,k) = 0.0_r8 
```

Comment:
If there isn't a GitHub issue regarding this, we should probably open an issue to look at it. I think it is important to know where these NaNs are coming from.

Reply:
I spent weeks tracing this down a couple years ago and got nowhere. I'll need to try taking this out and seeing if I can reproduce the problem. I forget the details, but I assume this was from the 2-mom case, which none of us are running now. And since we're planning to rip out the current 2-mom scheme in favor of the new P3 from SCREAM, maybe we should just take this out? 

The only potential problem I see with taking it out is that it might start breaking our 2-mom tests, and that could hinder our efforts to refactor ECPP. 

I'm not sure what to do here, curious to hear what others think.  
@brhillman @pressel @crjones-amath @lee1046

Reply:
@singhbalwinder I agree that there should be a Github issue for this if it's still a problem. However, my preference would be to leave this code in for the purpose of this PR in case it's still needed, and since the issue is only relevant to certain configurations of the MMF, then it doesn't become an issue in E3SM until this PR is merged anyways. So I think it makes the most sense to leave this in for now, merge the PR, and then confirm that removing this code still results in NaNs for the configurations that @whannah1 found before and then open an issue.

Reply:
Thanks. This check is within `use_MMF` if condition so it doesn't affect the default E3SM model run. If this block is needed for the time being, it can be left in the code.